app/src/main/kotlin/com/bitwarden/authenticator/ui/platform/base/util/StringExtensions.kt

```diff
@@ -72,3 +72,11 @@ fun String.removeDiacritics(): String =
             Normalizer.normalize(this, Normalizer.Form.NFKD),
             "",
         )
+
+/**
+ * Checks if a string is using base32 digits.
+ */
+fun String.isBase32(): Boolean {
+    val regex = ("^[A-Z2-7]+=*$").toRegex()
```

Comment:
🤔 While I know Base 32 is technically just uppercase characters, in practice people will use lowercase characters, so it probably makes sense to either make this case insensitive, or to uppercase the string before checking? (Or you already do that elsewhere)

Reply:
Good callout. I'll double check we're handling casing appropriately.